lib/Service/Sync/ImapToDbSynchronizer.php

```diff
@@ -232,7 +236,8 @@ public function sync(Account $account,
 		if (!$batchSync) {
 			$this->dispatcher->dispatchTyped(
 				new SynchronizationEvent(
-					$account
+					$account,
+					$this->logger
```

Comment:
All other usages in this method use `$logger` not `$this->logger`?

Reply:
Yeah because that is a different path, currently just triggered via a controller/service and not the CLI